src/vaadin-grid-sorter.html

```diff
@@ -120,6 +120,7 @@
              * How to sort the data.
              * Possible values are `asc` to use an ascending algorithm, `desc` to sort the data in
              * descending direction, or `null` for not sorting the data.
+             * @type {GridSorterDirection}
```

Comment:
Should probably have also `| undefined` to match (my suggestion of) the type of `direction` property in `vaadin-grid-sort-column` (where it is `undefined` by default). Also in this class there are `undefined` checks for this `direction` property so it should be good to use `@type {GridSorterDirection | undefined}` for this in both places.
```suggestion
             * @type {GridSorterDirection | undefined}
```

Reply:
Done.